### PR TITLE
Add note explaining the origin of the name "foyer"

### PR DESCRIPTION
**Pull request by ctk3b**
As on tin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ generating, syntactically correct input files for various simulation engines. Fo
 dissemination, helping to eliminate ambiguity in atom-typing and improving reproducibility
 (for more information, see [our paper](https://www.sciencedirect.com/science/article/pii/S0927025619303040) or its corresponding [pre-print](https://arxiv.org/pdf/1812.06779.pdf)).
 
+_About the name:_ foyer was inspired by the [antechamber](http://ambermd.org/antechamber/ac.html) tool in the [Amber](https://ambermd.org/) ecosystem. Foyer plays a similar role to antechamber, hence the tongue in cheek, shared vestibular reference.
+
```
Comment (mattwthompson) [inline README.md:18]:
```suggestion
_About the name:_ foyer was inspired by the [antechamber](http://ambermd.org/antechamber/ac.html) tool in the [Amber](https://ambermd.org/) ecosystem. Foyer plays a similar role to antechamber, hence the tongue-in-cheek, shared vestibular reference.

```

Nitpicky suggestion from somebody who submitted his doctoral thesis with a typo on the first line (and had to look up "vestibular")

**Review by mattwthompson — APPROVED** [review]
I believe @daico007 is at the wheel these days? I will let him hit the big green button

**Review by daico007 — APPROVED** [review]
Love the history lesson! My be we should add something about how we end up with the MoSDeF abbr.

**Comment by ctk3b** [comment]
I too had to look up the word vestibular 😁

**Pull request merged**